### ISSUE TITLE
Default `account_tags` variable for `aft-account-request` module not set

### ISSUE THREAD

**Issue by jeremyciak** [aws-ia/terraform-aws-control_tower_account_factory#60]
Can we please set the `account_tags` variable for the `aft-account-request` module to be an empty map (`{}`)?

https://github.com/aws-ia/terraform-aws-control_tower_account_factory/blob/main/sources/aft-customizations-repos/aft-account-request/terraform/modules/aft-account-request/variables.tf

**Comment by jeremyciak** [comment]
The use case was that I decided I didn't want to specify any tags and that I would evaluate the business requirements and specify some later on. Because this process is different than typical Terraform development of writing the code and testing with a plan locally and whatnot, I submitted my account request with the omission of tags. I had to find in the AFT pipeline that the request had failed because the tags were a required variable. I just think defaulting to an empty map allows for people to opt out of any tags more easily, should they choose to do so.
